.github/workflows/build.yml

```
@@ -0,0 +1,57 @@
+  pull_request:
+    branches: [main, development]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Contracts
+    if: >
+      !contains(github.event.pull_request.labels.*.name, 'ci: skip-build')
+    steps:
+      - uses: actions-rs/toolchain@v1
```

actionlint

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)